### ISSUE TITLE
CSTK Score is now TRUST Score... needs a full overhaul

### ISSUE THREAD

**Issue by GriffGreen** [commons-stack/swissmem-dapp#343]
Urgent:
- [x] Switch to TRUST Token monitoring instead of CSTK Token monitoring for updating the Token Holdings effectively Ctrl-f `0xc4fbE68522ba81a28879763C3eE33e08b13c499E` and replace with `0x0f478Fc765Ed7a5C52627B0acDFa351FB94309e4`
- [x] Ctrl-f "CSTK Score" and replace with "TRUST Score" Everywhere 
- [x] Ctrl-f "CSTK" and replace with "$TRUST" Everywhere else (after doing the check box before)
- [x] Ctrl-f "Commons Stack Association" and replace with "Trusted Seed" Everywhere 

Next in line (more complicated):
- [ ] Update the Terms and Statutes with the new Terms (reach out to @iviangita for those... Will also need to post to IPFS and linke from there)
- [x] Change logo EVERYWHERE (ask @kristoferlund for the logo)
- [x] Ctrl-f "Commons Stack" and replace with "Trusted Seed" Everywhere (except where it would sound weird like in the members page, where it says , "Members of the Trusted Seed put skin in the game by contributing to the Commons Stack, entitling them to a CSTK Score which denotes their reputation in the Trusted Seed." and it should just say, "Members of the Trusted Seed put skin in the game by contributing, entitling them to a TRUST Score which denotes their reputation in the Trusted Seed." There are probably a lot of weird ones like this.

Nice to have:
- [ ] Switch up the Token so we can start using the new Swag shop (BLOCKED by swag store)
- [x] Work with @mafer-cordovas to update the Members page
